.gitlab-ci.yml

```diff
@@ -201,3 +201,20 @@ clean_merged_prs:
   variables:
     DEPLOYMENT_BASE: $DEPLOYMENT_BASE
     GITHUB_PULL_REQUEST_ID: "merged"
+
+
+# some changes require a human to be notified. We'll do this through JIRA tickets
+notifications:
+  needs: [atomic_build]
+  image: python:3.11-bullseye
+  rules:
+    - if: '$CI_COMMIT_BRANCH == $DEPLOYMENT_DEFAULT_BRANCH'
+      changes:
+        - var/spack/repos/builtin/packages/apptainer/*
+        - var/spack/repos/builtin/packages/singularity/*
+        - var/spack/repos/builtin/packages/singularityce/*
+  script:
+    - pip install -r bluebrain/notifications/requirements.txt
+    - git log -n 3 >> bluebrain/notifications/apptainer_singularity.txt
+    - echo -e '{noformat}\n\nThank you!' >> bluebrain/notifications/apptainer_singularity.txt
```

Comment:
I'm wondering if we could use some formatting inside the template and add something like
```
{git_log}

Thank you!
```
there, then read the file and do `contents.format(git_log=subprocess.check_output(["git", ...])`?

Then it's all nicely contained in the Python script + template.

Would also allow to replace the git history beginning of the URL with some environment variable pointing to the Github repo